docs/modules/node.md

```diff
@@ -272,30 +276,68 @@ system heap size left in bytes (number)
 
 ## node.info()
 
-Returns NodeMCU version, chipid, flashid, flash size, flash mode, flash speed, and Lua File Store (LFS) usage statics.
+Returns information about hardware, software version and build configuration.
+
 
 #### Syntax
-`node.info()`
+`node.info([kind])`
```

Comment:
English is not my mother tongue but I guess I would have called this something like "group".

Reply:
Group sounds much better to me too.